tools/bin/load_test/load_test_utils.sh

```diff
@@ -0,0 +1,49 @@
+<<comment
+This file contains common util functions for use in load testing scripts.
+comment
+
+echo "Loading utils from $0"
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;34m'
+CLEAR='\033[0m'
+
+function callApi {
+  # call the API with the endpoint passed as arg $1, and (optional) payload passed as arg $2
+  # example of calling the API with a payload:
+  #    callApi "destinations/list" "{\"workspaceId\":\"${workspace}\"}"
+  endpoint=$1
+  if [ -z "${2-}" ] ; then
+    payload=""
+  else
+    payload=$2
+  fi
+
+  curl --silent \
```

Comment:
this looks so pretty on multi-line with long args 😍 
![giphy-10](https://user-images.githubusercontent.com/2591516/196732434-06e294ef-4dc0-49a5-a354-da1de920c015.gif)